docs/src/molecules/dropdownButton.jsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable global-require */
+/* eslint-disable import/no-webpack-loader-syntax */
+
+import {
+    Card,
+    Header,
+    TitleBar,
+} from 'react-cm-ui';
+import _ from 'lodash';
+import React from 'react';
+import DropdownButtonExample from './dropdownButtonExample';
+import Highlighter from '../global/highlighter';
+import Main from '../global/main';
+import TableProps from '../global/tableProps';
+import { buttonDocProps } from '../atoms/buttonConstants';
+
+function DocsDropdownButton() {
+    let tableProps = _.filter(buttonDocProps, (prop) => (
+        prop.name !== 'as' &&
+        prop.name !== 'icon' &&
+        prop.name !== 'href' &&
+        prop.name !== 'onClick'
+    ));
+    tableProps = [
+        ...tableProps,
+        {
+            name: 'as',
+            type: 'button',
+            default: 'button',
+            description: 'Can only be a button.',
+        }, {
+            name: 'iconType',
+            type: 'chevron-down<br />| plus',
+            default: 'chevron-down',
+            description: 'An element type to render as.',
```

Comment:
This description seems off for `iconType` prop (maybe it was accidentally copied and pasted from an `as` prop somewhere...?)

Reply:
Oh, you're right. I forgot to update it.

Reply:
fixed.